integrations/azure/main.py

```diff
@@ -0,0 +1,11 @@
+# noinspection PyUnresolvedReferences
+# ruff: noqa: F401
+from azure_integration import ocean
+from integration import cloud_event_validation_middleware_handler
+
+from port_ocean.context.ocean import ocean as port_ocean
+
+
+port_ocean.app.fast_api_app.middleware("azure_cloud_event")(
```

Comment:
I think you can add the middleware to ocean.router

Reply:
fastapi doesn't support it